pages/landing/hipaa/template/pricing/index.html

```diff
@@ -29,24 +29,79 @@
         description: Deploy dev and stage infrastructure today and start building your application. HIPAA compliance coming soon!
       - name: HIPAA-compliant reference architecture
         description: Assessed by qualified 3rd parties for peace of mind.
-      - name: HIPAA-compliant application templates
-        description: World-class application templates that you can easily extend.
+      - name: HIPAA-compliant sample applications
+        description: World-class sample applications that you can easily extend.
       - name: Infrastructure as Code Library
         description: Access our entire library to augment and extend your reference architecture.
       - name: Gruntwork customer support
         description: Gruntwork engineers can answer your technical questions and help you resolve problems if they come up.
-      - name: Access to our library of training videos to ramp up on tools and techniques
+      - name: Access to our library of training videos to ramp up on tools and techniques.
         description: Our training library is a collection of video courses to help you learn Terraform, Terragrunt, and more.
       - name: HITRUST CSF audit assistance
         description: After your architecture is deployed, Gruntwork will work with you and your HITRUST assessor to help your infrastructure achieve the HITRUST certification.
       - name: Landing Zone
         description: Set up secure AWS baselines for your AWS accounts, with centralized logging, cross-account access, encryption by default, and more.
       - name: CI/CD Pipelines
         description: Streamline your infrastructure as code deployments through a repeatable CI/CD pipeline.
+
+faq:
+  - question: What sort of AWS bill should I expect?
+    answer: |
+      <p>
+        Your AWS bill depends on a number of factors: which services you use, the instance types you pick, how much load you're dealing with, etc. That said, assuming you have access to the AWS Free Tier, here's a ballpark range for what your AWS bill will be if you deploy an app using infrastructure from the following subscription option:
+      </p>
+      <ul>
+        <li><strong>Gruntwork Lite</strong>: $0 / month</li>
+        <li><strong>Gruntwork Pro</strong>: $500 / month and up</li>
+        <li><strong>Gruntwork Pro</strong>: $2,000 / month and up</li>
```

Comment:
Should this be Gruntwork Enterprise?

Reply:
Good catch - I lifted this from the heroku page's pricing FAQ - so we may need to update it there as well - fixed in dde5ca1a